src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -23,6 +25,37 @@
         (ds/rename-columns {col-to-avg avg-name})
         (wds/rollup :mean avg-name [:gss-code :sex :age]))))
 
+(defn jumpoffyr-method-trend
+  "Takes in a dataset with historical data, a column name for a trend to be 
+  calculated for, a name for the trend col and years to start and end to
+  calculate the trend for. Returns a dataset with a new col for the calculated trend"
+  [historical-data trend-col trend-out start-yr-avg end-yr-avg]
+  (let [hist-earliest-yr (reduce min (ds/column historical-data :year))
+        hist-latest-yr (reduce max (ds/column historical-data :year))
+        start-yr (s/validate (s/pred #(>= % hist-earliest-yr)) start-yr-avg)
+        end-yr (s/validate (s/pred #(<= % hist-latest-yr)) end-yr-avg)
+        grouped-data (->> (i/query-dataset historical-data
+                                           {:year {:$gte start-yr
+                                                   :$lte end-yr}})
+                          (i/$group-by [:sex :gss-code :age]))
+        intercept (map (fn [[k v]]
```

Comment:
What's the meaning of "intercept" in this context? 
We're getting out regression coefficients (https://github.com/incanter/incanter/blob/master/modules/incanter-core/src/incanter/stats.clj#L2052) is it the same thing as "intercept"?